.github/workflows/release.yml

```
@@ -28,3 +28,48 @@ jobs:
         env:
           # Default github token should have enough permissions to make a release
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  build-and-push-container-image:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+          check-latest: true
+      - name: Set up Docker Buildx
```

nit: \n to be consistent
```suggestion
          check-latest: true

      - name: Set up Docker Buildx
```